Views/Lombiq.Privacy.ConsentBanner.cshtml

```diff
@@ -66,7 +69,7 @@ else
     <script depends-on="jQuery" at="Foot">
         jQuery(($) => {
             $('#@acceptButtonId').click(function () {
-                var antiForgeryToken = $('[name="__RequestVerificationToken"]').val();
+                var antiForgeryToken = '@requestToken';
```

Comment:
It won't cause a problem here, but please make it a practice to avoid passing a Razor variable in a script block at all. It doesn't get JS string escaped and it does get HTML escaped which can cause problems and it's easier to always do it safely than to evaluate on a case-by-case basis. [Use our extension methods from HelpfulLibraries to convert them into JSON strings.](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/1748369476/Markup+conventions+and+Razor+practices#:~:text=if%20you%20place%20it%20into%20a%20javascript%20string) That returns an already quoted `HtmlString`, so you won't need to surround the output with apostrophes.
Please do the same with the other instances in this file.